easee/charger.py

```diff
@@ -85,7 +93,12 @@ async def get_state(self, from_cache=False):
         if not from_cache:
             state = await (await self.easee.get(f"/api/chargers/{self.id}/state")).json()
             self._state = ChargerState(
-                {**state, "chargerOpMode": STATUS[state["chargerOpMode"]],}, self.easee
+                {
+                    **state,
+                    "chargerOpMode": STATUS[state["chargerOpMode"]],
+                    "reasonForNoCurrent_desc": REASON_FOR_NO_CURRENT.get(state["reasonForNoCurrent"], "Unknown"),
```

Comment:
I like it for chargerOpMode and the other ones, but I see from the cloud website that these text are quite long, so for this case I found it natural to show a fault code and a text separately. I guess it could be nice for UI later show the fault code with hovering for the text for instance. What do you think?

Reply:
I don't think an end user is interested in the integer codes. I prefer to have human readable only. At least I want it one way, not two.

Reply:
Understand. Let's keep it as you suggest until we see need for anything else.